health/egov-persister/plan-service-persister.yml

```diff
@@ -198,6 +198,63 @@ serviceMaps:
             - jsonPath: $.PlanConfiguration.auditDetails.lastModifiedBy
             - jsonPath: $.PlanConfiguration.auditDetails.lastModifiedTime
 
+    - version: 1.0
+      description: Persists the plan employee assignment details
+      fromTopic: plan-employee-assignment-create-topic
+      isTransaction: true
+      queryMaps:
+        - query: INSERT INTO health.plan_employee_assignment(id, tenant_id, plan_configuration_id, employee_id, role, jurisdiction, additional_details, active, created_by, created_time, last_modified_by, last_modified_time) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?);
+          basePath: $.PlanEmployeeAssignment
+          jsonMaps:
+            - jsonPath: $.PlanEmployeeAssignment.id
+
+            - jsonPath: $.PlanEmployeeAssignment.tenantId
+
+            - jsonPath: $.PlanEmployeeAssignment.planConfigurationId
+
+            - jsonPath: $.PlanEmployeeAssignment.employeeId
+
+            - jsonPath: $.PlanEmployeeAssignment.role
+
+            - jsonPath: $.PlanEmployeeAssignment.jurisdiction
+              type: STRING
```

Comment:
why is type STRING and dbType STRING needed? Remove this

Reply:
> why is type STRING and dbType STRING needed? Remove this

removed